nautobot_golden_config/tables.py

```diff
@@ -357,3 +358,52 @@ class Meta(BaseTable.Meta):
         model = models.ConfigReplace
         fields = ("pk", "name", "platform", "description", "regex", "replace")
         default_columns = ("pk", "name", "platform", "description", "regex", "replace")
+
+
+class GoldenConfigSettingTable(BaseTable):
+    # pylint: disable=R0903
+    """Table for list view."""
+
+    pk = ToggleColumn()
+    name = LinkColumn("plugins:nautobot_golden_config:goldenconfigsetting", args=[A("pk")])
+    jinja_repository = Column(
+        verbose_name="Jinja Repository", empty_values=(),
+    )
+    intended_repository = Column(
+        verbose_name="Intended Repository", empty_values=(),
+    )
+    backup_repository = Column(
+        verbose_name="Backup Repository", empty_values=(),
+    )
+
+    def _render_capability(self, record, column, record_attribute):
+        if getattr(record, record_attribute, None):
+            return "✔"
+        else:
+            return "✘"
+
+    def render_backup_repository(self, record, column):  # pylint: disable=no-self-use
```

Comment:
just to confirm my understanding, these methods `render_<name_field>` are used to render the output of these fields in the table?

Reply:
Correct